src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="body">
+    <div class="calendar calendar--start-day--mon calendar--month-length--31">
+      <div class="calendar__day calendar__day--1"></div>
```

Comment:
it should work like this
```suggestion
      <div class="calendar__day"></div>
```